### DISCUSSION TITLE
옵션 관리 코드 누락 관련

### DISCUSSION THREAD

**Discussion by leedors527** [commerce-api-naver/commerce-solutions#64] [묻고 답하기] [answered]
{
    "originProduct": {
        "statusType": "SALE",
        "saleType": "SALE",
        "leafCategoryId": "50003573",
        "name": " \uD1B0\uBCF4 \uD39C 2B \uAE68\uB057\uD558\uAC8C \uC0AC\uB77C\uC9C0\uB294  KB-KSKM01-2B",
        "detailContent": "\u003Chtml\u003E\u003Cbody\u003E\u003Ctable id=\u0027OptionTable\u0027 style=\u0027width:600px; border-collapse:collapse; text-align:center;\u0027\u003E\u003Ctr\u003E\u003Ctd colspan=\u00274\u0027 style=\u0027font-weight:bold; padding:10px;\u0027\u003E\uC635\uC158 \uC124\uBA85\u003C/td\u003E\u003C/tr\u003E\u003Ctr\u003E\u003Ctd style=\u0027border:1px solid #ddd; padding:8px;\u0027\u003E\u003Cimg src=\u0027https://m.media-amazon.com/images/I/41wjj5pYWzL._AC_.jpg\u0027 style=\u0027width:200px; height:200px;\u0027\u003E\u003C/td\u003E\u003Ctd style=\u0027border:1px solid #ddd; padding:8px;\u0027\u003E\u003Cdiv style=\u0027margin-bottom:10px;\u0027\u003E\u003Cspan style=\u0027display:block; font-weight:bold;\u0027\u003E\uC7AC\uACE0 \uC788\uC74C :   \u003C/span\u003E\u003C/div\u003E\u003C/td\u003E\u003C/tr\u003E\u003Ctr\u003E\u003Ctd style=\u0027border:1px solid #ddd; padding:8px;\u0027\u003E\u003Cimg src=\u0027https://m.media-amazon.com/images/I/41e9KEiFtkL._AC_.jpg\u0027 style=\u0027width:200px; height:200px;\u0027\u003E\u003C/td\u003E\u003Ctd style=\u0027border:1px solid #ddd; padding:8px;\u0027\u003E\u003Cdiv style=\u0027margin-bottom:10px;\u0027\u003E\u003Cspan style=\u0027display:block; font-weight:bold;\u0027\u003E\uC7AC\uACE0 \uC788\uC74C :   \u003C/span\u003E\u003C/div\u003E\u003C/td\u003E\u003C/tr\u003E\u003C/table\u003E\u003C/body\u003E\u003C/html\u003E",
        "images": {
            "representativeImage": {
                "url": "https://shop-phinf.pstatic.net/20241119_99/1731956218591irVRA_JPEG/45841497395387186_1818059006.jpg"
            },
            "optionalImages": [{
                    "url": "https://shop-phinf.pstatic.net/20241119_99/1731956218591irVRA_JPEG/45841497395387186_1818059006.jpg"
                }, {
                    "url": "https://shop-phinf.pstatic.net/20241119_23/1731956218736d8QKV_JPEG/45841497539528886_1671225992.jpg"
                }, {
                    "url": "https://shop-phinf.pstatic.net/20241119_141/1731956218867vlJTL_JPEG/45841497671907922_1349476082.jpg"
                }, {
                    "url": "https://shop-phinf.pstatic.net/20241119_242/1731956219072awV5D_JPEG/45841497881072889_1969171934.jpg"
                }, {
                    "url": "https://shop-phinf.pstatic.net/20241119_230/1731956219210p1j57_JPEG/45841498019816900_445740059.jpg"
                }, {
                    "url": "https://shop-phinf.pstatic.net/20241119_113/1731956219403PM4SW_JPEG/45841498206807540_1056030130.jpg"
                }, {
                    "url": "https://shop-phinf.pstatic.net/20241119_131/1731956219561DU0m0_JPEG/45841498366083583_1343464542.jpg"
                }, {
                    "url": "https://shop-phinf.pstatic.net/20241119_260/1731956216604VSo9f_JPEG/61965891273226319_1158616308.jpg"
                }, {
                    "url": "https://shop-phinf.pstatic.net/20241119_97/17319562197396f4yO_JPEG/45841498544540879_751533402.jpg"
                }
            ]
        },
        "saleStartDate": "2024-11-18T19:15:56Z",
        "saleEndDate": "2025-01-17T19:15:56Z",
        "salePrice": 88240,
        "stockQuantity": 1000,
        "deliveryInfo": {
            "deliveryType": "DELIVERY",
            "deliveryAttributeType": "NORMAL",
            "deliveryCompany": "CJGLS",
            "deliveryFee": {
                "deliveryFeeType": "PAID",
                "baseFee": 2000,
                "freeConditionalAmount": 0,
                "repeatQuantity": 0,
                "secondBaseQuantity": 0,
                "secondExtraFee": 0,
                "thirdBaseQuantity": 0,
                "thirdExtraFee": 0,
                "deliveryFeePayType": "PREPAID",
                "deliveryFeeByArea": null
            },
            "deliveryBundleGroupUsable": false,
            "claimDeliveryInfo": {
                "returnDeliveryCompanyPriorityType": "PRIMARY",
                "returnDeliveryFee": 2000,
                "exchangeDeliveryFee": 4000,
                "shippingAddressId": 103164951,
                "returnAddressId": 105271344,
                "freeReturnInsuranceYn": false
            }
        },
        "productLogistics": [],
        "detailAttribute": {
            "naverShoppingSearchInfo": null,
            "afterServiceInfo": {
                "afterServiceTelephoneNumber": "12345",
                "afterServiceGuideContent": "asds"
            },
            "purchaseQuantityInfo": {
                "minPurchaseQuantity": 2,
                "maxPurchaseQuantityPerId": 1000,
                "maxPurchaseQuantityPerOrder": 2
            },
            "originAreaInfo": {
                "originAreaCode": "00",
                "importer": "",
                "content": "",
                "plural": false
            },
            "sellerCodeInfo": {
                "sellerManagementCode": "UPN00000072",
                "sellerBarcode": "1234",
                "sellerCustomCode1": "1234",
                "sellerCustomCode2": "123456"
            },
            "optionInfo": {
                "optionCombinationSortType": "CREATE",
                "optionCombinationGroupNames": {
                    "optionGroupName1": "\uC0AC\uC774\uC988",
                    "optionGroupName2": "\uD0C0\uC785",
                    "optionGroupName3": "",
                    "optionGroupName4": null
                },
                "optionCombinations": [{
                        "id": null,
                        "optionName1": " \uB2E8\uC77C\uD488",
                        "optionName2": " 2B",
                        "optionName3": "",
                        "optionName4": null,
                        "stockQuantity": 1000,
                        "price": 0,
                        "sellerManagerCode": "UPO00000268",
                        "usable": true
                    }, {
                        "id": null,
                        "optionName1": " \uB2E8\uC77C\uD488",
                        "optionName2": " 4B",
                        "optionName3": "",
                        "optionName4": null,
                        "stockQuantity": 1000,
                        "price": 2900,
                        "sellerManagerCode": "UPO00000267",
                        "usable": true
                    }
                ],
                "standardOptionGroups": [],
                "standardOptionAttributes": [],
                "standardOptions": [],
                "optionDeliveryAttributes": []
            },
            "purchaseReviewInfo": {
                "purchaseReviewExposure": false,
                "reviewUnExposeReason": ""
            },
            "isbnInfo": {
                "isbn13": null,
                "issn": null,
                "independentPublicationYn": null
            },
            "bookInfo": null,
            "seoInfo": {
                "pageTitle": "asdg",
                "metaDescription": "sfsdf",
                "sellerTags": []
            },
            "releaseDate": null,
            "manufactureDate": "2024-09-20",
            "validDate": "2024-09-05",
            "taxType": "TAX",
            "certificationTargetExcludeContent": null,
            "minorPurchasable": false,
            "productInfoProvidedNotice": {
                "productInfoProvidedNoticeType": "BAG",
                "wear": null,
                "shoes": null,
                "bag": {
                    "type": "\uC0C1\uD488 \uC0C1\uC138\uC124\uBA85 \uCC38\uC870",
                    "material": "\uC0C1\uD488 \uC0C1\uC138\uC124\uBA85 \uCC38\uC870",
                    "color": "\uC0C1\uD488 \uC0C1\uC138\uC124\uBA85 \uCC38\uC870",
                    "size": "\uC0C1\uD488 \uC0C1\uC138\uC124\uBA85 \uCC38\uC870",
                    "manufacturer": "\uC0C1\uD488 \uC0C1\uC138\uC124\uBA85 \uCC38\uC870",
                    "caution": "\uC0C1\uD488 \uC0C1\uC138\uC124\uBA85 \uCC38\uC870",
                    "packDate": "",
                    "packDateText": "",
                    "warrantyPolicy": "\uC0C1\uD488 \uC0C1\uC138\uC124\uBA85 \uCC38\uC870",
                    "afterServiceDirector": "\uC0C1\uD488 \uC0C1\uC138\uC124\uBA85 \uCC38\uC870",
                    "returnCostReason": "",
                    "noRefundReason": "",
                    "qualityAssuranceStandard": "",
                    "compensationProcedure": "",
                    "troubleShootingContents": ""
                },
                "fashionItems": null,
                "sleepingGear": null,
                "furniture": null,
                "imageAppliances": null,
                "homeAppliances": null,
                "seasonAppliances": null,
                "officeAppliances": null,
                "opticsAppliances": null,
                "microElectronics": null,
                "navigation": null,
                "carArticles": null,
                "cellPhone": null,
                "medicalAppliances": null,
                "kitchenUtensils": null,
                "cosmetic": null,
                "jewellery": null,
                "food": null,
                "generalFood": null,
                "dietFood": null,
                "kids": null,
                "musicalInstrument": null,
                "sportsEquipment": null,
                "books": null,
                "rentalEtc": null,
                "digitalContents": null,
                "giftCard": null,
                "mobileCoupon": null,
                "movieShow": null,
                "etcService": null,
                "biochemistry": null,
                "biocidal": null,
                "etc": null
            }
        },
        "customerBenefit": {
            "immediateDiscountPolicy": {
                "discountMethod": null
            },
            "ourchasePointPolicy": null,
            "reviewPointPolicy": null,
            "freeInterestPolicy": {
                "value": 0,
                "startDate": "",
                "endDate": ""
            },
            "giftPolicy": {
                "presentContent": "sgsg"
            },
            "multiPurchaseDiscountPolicy": null,
            "reservedDiscountPolicy": {
                "discountMethod": null
            }
        }
    },
    "smartstoreChannelProduct": {
        "channelProductName": null,
        "bbsSeq": null,
        "storeKeepExclusiveProduct": false,
        "naverShoppingRegistration": false,
        "channelProductDisplayStatusType": "ON"
    }
}

안녕하세요. 위와 같이 옵션의 관리 코드가 포함된 데이터를 상품 전송, 수정 API 에 전송했지만 관리 코드가 누락됩니다.

![image](https://github.com/user-attachments/assets/a8ac2021-6c60-4293-9b6a-c49be2a10d52)

확인 부탁드립니다. 감사합니다.

**Comment by commerce-api-naver** [comment, marked as answer]
안녕하세요. 커머스API 이민석입니다.

대상하는 호출건의 Trace ID를 알려주실 수 있을까요?
Trace ID는 API 호출에 따른 Response header(GNCP-GW-Trace-ID) 또는 Response body(일부 API에 한함)로 제공됩니다.

**Reply by leedors527** [reply]
별도로 작업한 게 없는데 지금은 또 되네요. 답변 감사합니다.
